src/Autofac/Core/Resolving/Pipeline/ResolveRequestContextBase.cs

```diff
@@ -162,21 +162,10 @@ public void ChangeParameters(IEnumerable<Parameter> newParameters)
         }
 
         /// <inheritdoc />
-        public abstract object ResolveComponent(ResolveRequest request);
-
-        /// <summary>
-        /// Resolve an instance of the provided registration within the context, but isolated inside a new
-        /// <see cref="ResolveOperationBase"/>.
-        /// This method should only be used instead of <see cref="IComponentContext.ResolveComponent(ResolveRequest)"/>
-        /// if you need to resolve a component with a completely separate operation and circular dependency verification stack.
-        /// </summary>
-        /// <param name="request">The resolve request.</param>
-        /// <returns>
-        /// The component instance.
-        /// </returns>
-        /// <exception cref="ComponentNotRegisteredException"/>
-        /// <exception cref="DependencyResolutionException"/>
-        public abstract object ResolveComponentWithNewOperation(ResolveRequest request);
+        public object ResolveComponent(ResolveRequest request)
```

Comment:
Given this the base class should this be `virtual`?

Reply:
Yes, that makes sense; will sort that out tomorrow.